#
# Submitted-By: Luotao Fu, 2006-09-18
# Committed-By: Luotao Fu
#
# Error:
#
# Bash compilation fails at variables.c 
#
# Description:
#
# winsize_assignment is only declared while READLINE is set, however the ifdef 
# is missing on one place before referencing this variable, which causes 
# compilation error. Added ifdef here to fix it
#
# State:
# 
# FIXED
#
Index: bash-3.1/variables.c
===================================================================
--- bash-3.1.orig/variables.c
+++ bash-3.1/variables.c
@@ -860,9 +860,11 @@ sh_set_lines_and_columns (lines, cols)
 {
   char val[INT_STRLEN_BOUND(int) + 1], *v;
 
+#if defined (READLINE)
   /* If we are currently assigning to LINES or COLUMNS, don't do anything. */
   if (winsize_assignment)
     return;
+#endif /*READLINE*/
 
   v = inttostr (lines, val, sizeof (val));
   bind_variable ("LINES", v, 0);
